library/std/src/os/uefi/raw.rs

```diff
@@ -0,0 +1,3 @@
+//! This module just re-exports stuff from r-efi crate
+
+pub use r_efi::efi::{BootServices, Guid, RuntimeServices, SystemTable};
```

Comment:
Having read through more of r-efi I am confident this is not something we do not want to do.

Reply:
So should I redeclare them under `std::os::uefi::raw`? Or should I just return a `void` pointer from the apis under `std::os::uefi::env`?

Reply:
You should use c_void if you need these in public APIs I guess. There are other options and it's t-libs-api's decision, but exposing them from std is absolutely not an option. You can use them internally, though.

Reply:
I am returning void pointers now. I have also updated the API change proposal. Feel free to reopen the conversation.